docs/release-notes/ngc-images.rst

```diff
@@ -0,0 +1,18 @@
+:orphan:
+
+**New Features**
+
+-  Add Add early access NVIDIA NGC-based images to our environments offerings. These images can be
+   pulled from `https://hub.docker.com/r/determinedai/pytorch-ngc` or
+   `https://hub.docker.com/r/determinedai/tensorflow-ngc`. By downloading and using these images,
+   you accept the terms and conditions of all third-party software licenses contained within,
+   including the (`NVIDIA Deep Learning Container
+   License.<https://developer.download.nvidia.com/licenses/NVIDIA_Deep_Learning_Container_License.pdf>`_)
+   Users can build their own images from a given NGC container version by using the
+   `build-pytorch-ngc` or `build-tensorflow-ngc` workflows found in our environments `MakeFile`. Our
+   environments repo is located at `https://github.com/determined-ai/environments`.
+
```

Comment:
:orphan:

**New Features**

-  Include early-access NVIDIA NGC-based images in our environment offerings. These images are accessible from `https://hub.docker.com/r/determinedai/pytorch-ngc` or
   `https://hub.docker.com/r/determinedai/tensorflow-ngc`. By downloading and using these images, users acknowledge and agree to the terms and conditions of all third-party software licenses contained within, including the `NVIDIA Deep Learning Container License
<https://developer.download.nvidia.com/licenses/NVIDIA_Deep_Learning_Container_License.pdf>`__.
   Users can build their own images from a specified NGC container version by using the
   ``build-pytorch-ngc`` or ``build-tensorflow-ngc`` workflows located in our environments ``MakeFile``. The environments repository is located at `https://github.com/determined-ai/environments`.

Reply:
wow, it's nearly impossible to show "restructured text markup" in a "markdown" comment

Reply:
`:orphan:

**New Features**

-  Include early-access NVIDIA NGC-based images in our environment offerings. These images are
   accessible from `pytorch-ngc <https://hub.docker.com/r/determinedai/pytorch-ngc>`_ or
   `tensorflow-ngc <https://hub.docker.com/r/determinedai/tensorflow-ngc>`_. By downloading and
   using these images, users acknowledge and agree to the terms and conditions of all third-party
   software licenses contained within, including the `NVIDIA Deep Learning Container License
   <https://developer.download.nvidia.com/licenses/NVIDIA_Deep_Learning_Container_License.pdf>`__.
   Users can build their own images from a specified NGC container version by using the
   ``build-pytorch-ngc`` or ``build-tensorflow-ngc`` workflows located in our environments
   ``MakeFile`` in the `environments repository <https://github.com/determined-ai/environments>`_.
`

Reply:
applied a commit that fixes the formatting and modifies some word choices